lte/gateway/python/magma/kernsnoopd/snooper.py

```diff
@@ -0,0 +1,107 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import os
+
+from bcc import BPF
```

Comment:
Should this be guarded? Log and terminate service if bcc not installed.

Reply:
What is the dependency to have BPF here? Is it `bpfcc-tools` mentioned in build-magma.sh?

Reply:
Yes. _Snooper_ needs bcc to compile and load eBPF programs to kernel.